src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -2083,6 +2383,30 @@ bool WholeBodyDynamicsDevice::resetOffset(const std::string& calib_code)
     return true;
 }
 
+bool WholeBodyDynamicsDevice::usePreEstimatedOffset(const std::string& calib_code)
+{
+    yarp::os::LockGuard guard(this->deviceMutex);
+
+    yWarning() << "wholeBodyDynamics : calib ignoring calib_code " << calib_code << " using offset estimated offline.";
+
+    for(size_t ft = 0; ft < this->getNrOfFTSensors(); ft++)
+    {
+        double norm = iDynTree::toEigen(ftProcessors[ft].estimatedOffset()).norm();
+        if (norm!= 0.0){ // if norm is exactly 0.0 it means there was no offset  in the configuration file so it should be skipped
+
+            //The offset estimated offline is meant to be added, while it is substracted in the six axis force torque  measure helpers, so we changed the sign here. There might be a better solution though.
```

Comment:
Can't we just change the convention of the FT offset parameter to be consistent with what is already there? These logic is actually quite old, and changing convention of FT offset just for some parameters is quite error while keeping the old one in other cases (for example when we print the offset) is extremely error prone.